src/precompiles/ecrecover.rs

```diff
@@ -18,11 +18,13 @@ pub use zkevm_opcode_defs::sha3::Keccak256;
 
 use super::*;
 
+const NUM_ROUNDS: usize = 1;
```

Comment:
We have this constant defined for two different precompiles, but it has the same value in both cases. We could move it to the `statistics.rs` file and rename it to something like `DEFAULT_PRECOMPILE_NUM_ROUNDS` to have it in one place.